src/data/fetchGeographyData.ts

```diff
@@ -0,0 +1,34 @@
+import * as dsv from "d3-dsv"; // https://github.com/d3/d3/issues/3469
+import { selectedGeographyStore } from "../stores/stores";
+
+const apiBaseUrl = `https://cep5lmkia0.execute-api.eu-west-1.amazonaws.com/dev`;
+
+export const fetchGeographyData = async (args: { totalCode: string; categoryCodes: string[]; geoCode: string }) => {
+  let data = await fetchSelectedGeographyData(args);
+  let parsedData = parseSelectedGeographyData(data, args.totalCode);
+  selectedGeographyStore.set({
+    displayName: "England and Wales",
```

Comment:
Are you intended to extend this with the name from the `geo` endpoint? If not, can I?

Reply:
Yep, exactly - it will mirror the fetchVizData function almost exactly